libs/perun/services/src/lib/route-policy.service.ts

```diff
@@ -99,9 +120,263 @@ export class RoutePolicyService {
+    ],
+    [
+      'resources-members',
+      (resource): boolean =>
+        this.authResolver.isAuthorized('getAssignedMembers_Resource_policy', [resource]),
+    ],
+    [
+      'resources-tags',
+      (resource): boolean =>
+        this.authResolver.isAuthorized('getAllResourcesTagsForResource_Resource_policy', [
+          resource,
+        ]),
+    ],
+    [
+      'resources-attributes',
+      (resource): boolean =>
+        this.authResolver.isAuthorized('getRichResourceById_int_policy', [resource]),
+    ],
+    [
+      'resources-settings',
+      (resource): boolean => this.authResolver.isManagerPagePrivileged(resource),
+    ],
+    [
+      'resources-settings-managers',
+      (resource): boolean => this.authResolver.isManagerPagePrivileged(resource),
+    ],
+    // Facilities
+    [
+      'facilities', // route for /facilities/:facilityId
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getFacilityById_int_policy', [facility]),
+    ],
+    [
+      'facilities-resources',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getAssignedRichResources_Facility_policy', [facility]),
+    ],
+    [
+      'facilities-allowed-users',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getAssignedUsers_Facility_Service_policy', [facility]),
+    ],
+    [
+      'facilities-allowed-groups',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getAllowedGroups_Facility_Vo_Service_policy', [facility]),
+    ],
+    [
+      'facilities-services-status',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getFacilityServicesState_Facility_policy', [facility]),
+    ],
+    [
+      'facilities-services-destinations',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getAllRichDestinations_Facility_policy', [facility]),
+    ],
+    // TODO fix when policies are updated
+    ['facilities-hosts', (): boolean => this.authResolver.isFacilityAdmin()],
+    [
+      'facilities-attributes',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getFacilityById_int_policy', [facility]),
+    ],
+    [
+      'facilities-settings',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getOwners_Facility_policy', [facility]) ||
+        this.authResolver.isAuthorized(
+          'getRichAdmins_Facility_List<String>_boolean_boolean_policy',
+          [facility]
+        ) ||
+        this.authResolver.isAuthorized('getAssignedSecurityTeams_Facility_policy', [facility]) ||
+        this.authResolver.isAuthorized('getBansForFacility_int_policy', [facility]),
+    ],
+    [
+      'facilities-settings-owners',
+      (facility): boolean =>
+        this.authResolver.isAuthorized('getOwners_Facility_policy', [facility]),
+    ],
+    [
+      'facilities-settings-managers',
+      (facility): boolean =>
+        this.authResolver.isAuthorized(
```

Comment:
Here and also in the settings we should use the `isManagerPagePrivileged` method from `GUIAuthResolver`.